src/javascripts/vendors.js

```diff
@@ -12,4 +12,4 @@ require('angular-numeraljs');
 require('angular-ui-bootstrap/dist/ui-bootstrap-tpls');
 require('ng-file-upload');
 
-global._ = require('underscore');
+global._ = require('lodash');
```

Comment:
..actually just tested, and if I reduce this to lodash.debounce, it is ok for the core code of ng-admin but it then make all Restangular interceptor code to fail as specified in the doc...
So not sure how this should be handled